src/babylon.node.ts

```diff
@@ -418,7 +418,7 @@
                 return;
             }
 
-            this._scene.beginAnimation(this, range.from, range.to, loop, speedRatio, onAnimationEnd);
+            return this._scene.beginAnimation(this, range.from, range.to, loop, speedRatio, onAnimationEnd);

```

Comment:
you will need to change the function's signature for that to work correctly.
see - https://travis-ci.org/BabylonJS/Babylon.js/builds/332791097